content/doc/book/installing/_kubernetes.adoc

```diff
@@ -0,0 +1,524 @@
+==== Installing Jenkins on a Kubernetes Cluster Using Helm v3
+
+A typical Jenkins deployment consists of a controller node and, optionally, one or more agents. To simplify the deployment of Jenkins, we’ll use link:https://helm.sh/[Helm] to deploy Jenkins. 
+Helm is a package manager for Kubernetes and its package format is called a chart. 
+Many community-developed charts are available on link:https://github.com/helm/charts[GitHub].
+
+Helm Charts provide “push button” deployment and deletion of apps, making adoption and development of Kubernetes apps easier for those with little container or microservices experience.
+
+===== Prerequisites
+
+. Helm CLI. If you don't have Helm CLI installed and configured locally, see the “Installing Helm and Setting Up Helm” sections below.
+
+===== Installing Helm
+
+To install Helm CLI, follow the instructions from the link:https://helm.sh/docs/intro/install/[Installing Helm] page.
+
+===== Setting Up Helm
+Once Helm is installed and set up properly, add the Jenkinsci repo as follows:
+
+[source,bash]
+----
+helm repo add jenkinsci https://charts.jenkins.io
+----
+
+[source,bash]
+----
+helm repo update
+----
+
+You can then run `helm search repo jenkinsci` to see the charts.
+
+===== Create a persistent volume for the Jenkins master
+
+We want to create a link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[persistent volume] for our Jenkins master pod. This will prevent us from losing our whole configuration of the Jenkins master and our jobs when we reboot our minikube. This link:https://github.com/kubernetes/minikube/blob/master/docs/persistent_volumes.md[official minikube doc] explains which directories we can use to mount or data. In a multi-node Kubernetes cluster, you’ll need some solution like NFS to make the mount directory available in the whole cluster. But because we use minikube which is a one-node cluster we don’t have to bother about it.
+
+We choose to use the /data directory. This directory will contain our Jenkins master configuration.
+
+*We will create a volume which is called jenkins-pv:*
+
+. Paste the following snippet into a file called jenkins-volume.yaml:
++
+[source,bash]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: jenkins-pv
+  namespace: jenkins-project
+spec:
+  storageClassName: jenkins-pv
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 20Gi
+  persistentVolumeReclaimPolicy: Retain
+  hostPath:
+    path: /data/jenkins-volume/
+----
+
+. Run the following command to apply the spec:
++
+[source,bash]
+----
+kubectl apply -f jenkins-volume.yaml
+----
+
+[NOTE]
+====
+
+It’s worth noting that, in the above spec, hostPath uses the /data/jenkins-volume/ of your node to emulate network-attached storage. This approach is only suited for development and testing purposes. For production, you should provide a network resource like a Google Compute Engine persistent disk, or an Amazon Elastic Block Store volume.
```

Comment:
+1 on including this.  If a lot of this is quickstart, would it make sense to break the quickstart piece to the beginning and have a section on "recommendations for running in production"?  I don't know if that would make more sense or be confusing.

Reply:
@kwhetstone do you mean removing this information and putting it somewhere else?

Reply:
I like that you included a message about what we're doing for the quickstart, and recommendations for production.  I think it's great to have these suggestions (production vs what we're doing for quickstart) for other sections.

Format-wise, I don't know the best way to highlight these great tips.  Would it be to pull out into a different section or to highlight them somehow throughout the tutorial?